README.md

```diff
@@ -0,0 +1,1221 @@
+    * [Methods of PreparedStatement object](#methods-of-preparedstatement-object)
+  * [Batch](#batch)
+    * [Methods of Batch object](#methods-of-batch-object)
+  * [Result](#result)
+    * [Methods of Result object](#methods-of-result-object)
+  * [Row](#row)
+    * [Methods of Row object](#methods-of-row-object)
+  * [Examples](#examples)
+    * [Basic usage](#basic-usage)
+    * [Binding Parameters](#binding-parameters)
+      * [Non Prepared Statement](#non-prepared-statement)
+      * [Prepared Statement](#prepared-statement)
+    * [Use prepared statement and paging](#use-prepared-statement-and-paging)
+    * [Configure Shard-Awareness connection](#configure-shard-awareness-connection-to-scylladb-cluster)
+    * [SSL Example](#ssl-example)
+    * [Retrieving metadata](#retrieving-metadata)
+    * [Configure logging](#configure-logging)
+      * [Set log level](#set-log-level)
+      * [Set callback for capture log messages](#set-callback-for-capture-log-messages)
+    * [Execution profiles](#execution-profiles)
+    * [Tracing](#tracing)
+  * [Developing](#developing)
+  
+
+## Features
+
+* Shard-Awareness
+* Asynchronous API
+* Simple, Prepared, and Batch statements
+* Asynchronous I/O, parallel execution, and request pipelining
+* Connection pooling
+* Automatic node discovery
+* Automatic reconnection
+* Configurable load balancing
+* Works with any cluster size
+* Authentication
+* SSL
+* Latency-aware routing
+* Performance metrics
+* Tuples and UDTs
+* Nested collections
+* Retry policies
+* Client-side timestamps
+* Data types
+* Idle connection heartbeats
+* Support for materialized view and secondary index metadata
+* Support for clustering key order, `frozen<>` and Cassandra version metadata
+* Whitelist/blacklist DC, and whitelist/blacklist hosts load balancing policies
+* Custom authenticators
+* Reverse DNS with SSL peer identity verification support
+* Randomized contact points
+* Speculative execution
+
+## Compatibility
+
+This driver works exclusively with the Cassandra Query Language v3 (CQL3) and
+Cassandra's native protocol. The current version works with:
+
+* Scylla and Scylla Enterprise
+* Apache Cassandra® versions 2.1, 2.2 and 3.0+
+* Python 3.7, 3.8, 3.9, 3.10 and 3.11 for Linux and MacOS 
+
+## Install
+
+There is an Beta realease compabitble with Python 3.7, 3.8, 3.9, 3.10 and 3.11 for Linux and MacOS environments uploaded as a Pypi package. Use the following
+command for installing it:
+
+```bash
+pip install acsylla
+```
+
+### Build your own package
+You can build your own package for any supported python version for ***x86_64*** and ***aarch64*** Linux.
+
+Example for build wheel for Python 3.11 aarch64 from master branch
+```bash
+curl -O https://raw.githubusercontent.com/acsylla/acsylla/master/bin/build.sh
+curl -O https://raw.githubusercontent.com/acsylla/acsylla/master/bin/build_in_docker.sh
+chmod +x build.sh build_in_docker.sh
+./build_in_docker.sh 3.11 master aarch64
+```
+
+## Cluster
```

Comment:
Any chance to move all deep documentation under its own `doc/documentation.md` file and leave into the Readme a basic snippet?

Reply:
Yes of course when we add publishing docs to readthedocs